test_scripts/Smoke/Resumption/ATF_Resumption_disconnect_more_than_30s_before_SUSPEND.lua

```diff
@@ -0,0 +1,124 @@
+--  Requirement summary:
```

Comment:
@LitvinenkoIra, this script is very common to ATF_Resumption_disconnect_more_than_30s_before_IGN_OFF, please clarify.